Cargo.toml

```
@@ -62,8 +62,8 @@ clap = { version = "4.1.6", default-features = false, features = [
   "cargo",
 ] }
 dashmap = { version = "5.4.0", features = ["serde"] }
-datafusion = { version = "21.0", features = ["simd"] }
-datafusion-common = "21.0"
+datafusion = { version = "21.1.0", features = ["simd"] }
+datafusion-common = "21.1.0"
```

[optional]
Consider omitting the “patch” version. You'll be able to update `Cargo.lock` with `cargo update -p datafusion` without editing `Cargo.toml`.
```suggestion
datafusion = { version = "21.1", features = ["simd"] }
datafusion-common = "21.1"
```

---

you are right